test/Core.Test/AdminConsole/OrganizationFeatures/Organizations/InitPendingOrganizationCommandTests.cs

```diff
@@ -15,10 +18,25 @@ namespace Bit.Core.Test.AdminConsole.OrganizationFeatures.Organizations;
 [SutProviderCustomize]
 public class InitPendingOrganizationCommandTests
 {
+
+    private readonly IOrgUserInviteTokenableFactory _orgUserInviteTokenableFactory = Substitute.For<IOrgUserInviteTokenableFactory>();
+    private readonly IDataProtectorTokenFactory<OrgUserInviteTokenable> _orgUserInviteTokenDataFactory = new FakeDataProtectorTokenFactory<OrgUserInviteTokenable>();
+
     [Theory, BitAutoData]
-    public async Task Init_Organization_Success(Guid userId, Guid orgId, Guid orgUserId, string publicKey,
-            string privateKey, SutProvider<InitPendingOrganizationCommand> sutProvider, Organization org)
+    public async Task Init_Organization_Success(User user, Guid orgId, Guid orgUserId, string publicKey,
+            string privateKey, SutProvider<InitPendingOrganizationCommand> sutProvider, Organization org, OrganizationUser orgUser)
     {
+        sutProvider.SetDependency(_orgUserInviteTokenDataFactory, "orgUserInviteTokenDataFactory");
+        sutProvider.Create();
+
+        _orgUserInviteTokenableFactory.CreateToken(orgUser).Returns(new OrgUserInviteTokenable(orgUser)
+        {
+            ExpirationDate = DateTime.UtcNow.Add(TimeSpan.FromDays(5))
+        });
+
+        var token = CreateToken(orgUser);
+        sutProvider.GetDependency<IOrganizationUserRepository>().GetByIdAsync(orgUserId).Returns(orgUser);
```

Comment:
I suggest you move the common token setup code into a private method. It's copy/pasted 6 times at the moment, and any "make the token check pass" is going to be a pretty common requirement for any new tests for future functionality.